packages/fx-core/tests/plugins/resource/function/unit/scaffold.test.ts

```diff
@@ -115,22 +144,5 @@ describe(FunctionPluginInfo.pluginName, () => {
       // Assert
       chai.assert.isTrue(ret.isOk());
     });
-
-    it("Test scaffold with fallback in TS", async () => {
```

Comment:
Remove the ts version test because the templates file is mock, so language does not matter o n this test any more.